src/styles/blocks/product.scss

```diff
@@ -0,0 +1,51 @@
+@use '../utils/mixins' as *;
+@use '../utils/variables' as *;
+
+.product {
+  border: 1px solid $gray-border;
+  border-radius: 5px;
+  display: flex;
+  flex-direction: column;
+  height: 408px;
```

Comment:
[Checklist](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules#css): Don't fix container size. Let the content size dictate it.